features/admin.applications.v1/components/forms/inbound-oidc-form.tsx

```diff
@@ -579,6 +584,34 @@ export const InboundOIDCForm: FunctionComponent<InboundOIDCFormPropsInterface> =
 
     }, [ initialValues ]);
 
+    /**
+     * Sets if subject token is enabled.
+     */
+    useEffect(() => {
+        setIsSubjectTokenEnabled(initialValues?.subjectToken ? initialValues?.subjectToken?.enable : false);
+    }, [ initialValues ]);
+
+    /**
+     * Sets if subject token feature is available.
+     */
+    useEffect(() => {
+        setIsSubjectTokenFeatureAvailable(initialValues?.subjectToken ? true : false);
+    }, [ initialValues ]);
+
+    /**
+     * Sets if subject token is enable button should be visible.
+     */
+    useEffect(() => {
+        if (isGrantChanged) {
+            if (!selectedGrantTypes?.includes("urn:ietf:params:oauth:grant-type:token-exchange")) {
```

Comment:
Let's use a constant for `"urn:ietf:params:oauth:grant-type:token-exchange"`. A constant for this should already have been defined in 	`application-management-constants` file.